missions/ME_000_Westphalian_Minors_Missions.txt

```diff
@@ -284,6 +284,9 @@ westphalian_minor_missions_missions_4 = {
 		position = 1
 		trigger = {
 			workshop = 3
```

Comment:
`
OR = {
    workshop = 3
    production_lvl_2 = 3
    counting_house = 3
    production_lvl_3 = 3
}
`

Reply:
While still not perfect, this is what you want in here I think.